.scripts/prepublish.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+const babel = ['node_modules', '.bin', 'babel'].join(path.sep);
+
+require('./ver');
+
+
+const args = [
+  '--ignore tests,__tests__,stories,story.jsx,story.js',
+  '--plugins "transform-runtime"',
+  './src --out-dir ./dist',
+].join(' ');
+
+const cmd = `${babel} ${args}`;
+shell.echo(chalk.gray(cmd));
```

Comment:
Sorry if I misunderstood, do you mean output like this:
![image](https://cloud.githubusercontent.com/assets/14885189/24846228/118070d2-1de3-11e7-951f-de0d84d269d2.png)

(this echo just mirrows executed command to keep it clear to developers what is going on here)